src/Autocomplete/doc/index.rst

```diff
@@ -159,6 +159,34 @@ After creating this class, use it in your form:
     these won't be used during the Ajax call to fetch results. Instead, include
     all options inside the custom class (``FoodAutocompleteField``).
 
+.. note::
+
+    .. versionadded:: Unreleased
+    The default route for Autocomplete components is ``/autocomplete/{alias}/``
+    Sometimes it may be useful to customize this URL - e.g. so that the component lives
+    under a specific firewall.
+
+    To use another route, you need to declare it :
+
+    .. code-block:: yaml
```

Comment:
Add an empty line between `code-block` and the actual code - that's just something we always do